benchexec/tools/ceagle.py

```diff
@@ -24,19 +25,8 @@
 
 class Tool(benchexec.tools.template.BaseTool):
 
-    REQUIRED_PATHS = [
```

Comment:
Could please keep and update the `REQUIRED_PATHS` list? It contains all paths that are part of your tool and are necessary to execute.